src/SFRField.cpp

```diff
@@ -51,3 +51,13 @@ bool SFRInterface::getRestore()
 {
     return restore;
 }
+
+void SFRInterface::setDefaultValue(int val)
+{
+    default_value = val;
```

Comment:
I may be misunderstanding, but do we actually need to include a setter function for default_value parameter since a changed default_value wouldn't be saved across reboot? (unless the default value actually is saved to eeprom in addition to the current value)

Reply:
The setter function is to set the default_value member for SFRInterface objects since they are private members. The setter function is used only in SFRField constructors. The default_value member is used only for testing. Alternatively, they could be changed to protected fields.

Reply:
Fixed to protected fields bc it's cleaner.

Reply:
Ah got it, I missed that first glance